services/storm/Dockerfile

```diff
@@ -31,6 +31,6 @@ RUN    wget -q https://archive.apache.org/dist/storm/apache-storm-1.1.0/$PACKAGE
 
 WORKDIR /opt/storm/
 
-COPY conf/storm.yaml /opt/storm/conf/storm.yaml
```

Comment:
why do you change this?

Reply:
Because in other case I had to create .force-dir-exists file into service/storm/config directory.